### DISCUSSION TITLE
Welcome to multi-logger-python-discord-token-logger-and-chrome-password-stealer-through-webhooks Discussions!

### DISCUSSION THREAD

**Discussion by Napoleon-x** [Napoleon-x/multi-logger-python-discord-token-logger-and-chrome-password-stealer-through-webhooks#1] [Announcements]
Are there any bugs or issues that we should fix?

**Comment by ghost** [comment]
your mom

**Reply by Napoleon-x** [reply]
Please refrain from being canadian

**Reply by boneyface1234** [reply]
Im struggling to figure out how to install the pip install -r requirements.txt as im decently new to coding/python overall '

- Fixed
